linear_x_vel_max: 1.1  # Max velocity on Linear X
linear_x_vel_min: 0.5  # min velocity on Linear X
linear_y_vel_max: 1.1  # Max velocity on Linear Y
linear_y_vel_min: 0.5  # min velocity on Linear Y
linear_z_vel_max: 3.14 # Max velocity on Angular Z (PI)
linear_z_vel_min: 1.04 # min velocity on Angular Z (PI/3)
holonomic: true        # enable holonomic robot or not
